.clj-kondo/config.edn

```diff
@@ -657,14 +659,19 @@
  ;; this list isn't exhaustive because it misses some stuff like the test runner and HTTP client but it's easier to go
  ;; fix those namespaces than it is to make this regex super hairy.
  [{:pattern "(?:.*-test(?:\\..*)?$)|(?:.*test-util$)|(?:^metabase\\.test.*)"
-   :name    test-namespaces}]
+   :name    test-namespaces}
+  {:pattern "metabase\\.cmd.*"
+   :name    printable-namespaces}]
 
  :config-in-ns
  {test-namespaces
   {:linters
-   {:inline-def        {:level :off}
-    :missing-docstring {:level :off}
-    :private-call      {:level :off}
+   {:inline-def                                    {:level :off}
+    :missing-docstring                             {:level :off}
+    :private-call                                  {:level :off}
 
     ;; custom linters
```

Comment:
Note that I'd like to fine-grain this for metabase.cmd nses to not turn all the discouraged vars off, but kondo doesn't support this. I did put in a [kondo ticket](https://github.com/clj-kondo/clj-kondo/issues/1979) for this.